validators/validation.js

```diff
@@ -0,0 +1,15 @@
+const Joi = require("joi");
+const pattern = /[`!@#$%^&*()_+\-=\[\]{};':"\\|,.<>\/?~][0-9]/;
+exports.signUpSchema = Joi.object({
+  email: Joi.string().email().lowercase().required(),
+  password: Joi.string().pattern(new RegExp(pattern)).min(3).max(15).required(),
```

Comment:
Nice use of regular expression. Hope you know what the constraint is and hope you have validated the constraint?